tests/TestCase/Core/StaticConfigTraitTest.php

```diff
@@ -85,20 +87,19 @@ public function testSetConfigValues(): void
     public function testGetConfigOrFail(): void
     {
         $className = get_class($this->subject);
-        $className::setConfig('foo', ['bar' => true]);
+        $className::setConfig('foo2', ['bar' => true]);
 
-        $result = $className::getConfigOrFail('foo');
+        $result = $className::getConfigOrFail('foo2');
         $this->assertSame(['bar' => true], $result);
     }
 
     public function testGetConfigOrFailException(): void
     {
         $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessage('Expected configuration `foo` not found.');
+        $this->expectExceptionMessage('Expected configuration `unknown` not found.');
```

Comment:
Since we are now using the same anonymous class for every test I had to adjust some keys in here